lib/Doctrine/ODM/PHPCR/Configuration.php

```diff
@@ -383,3 +385,5 @@ public function getUuidGenerator()
             };
     }
 }
+
+interface_exists(MappingDriver::class);
```

Comment:
is this line on purpose?

Reply:
Yes, see the linked blog post

Reply:
sorry, i read the link only afterward and commented on my comment, but forgot to close this.